easybuild/easyconfigs/d/DGrid/DGrid-5.1-intel-2019a.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = "MakeCp"
+
+name = 'DGrid'
+version = '5.1'
+
+homepage = 'https://www2.cpfs.mpg.de/~kohout/dgrid.html'
+description = """DGrid computes and evaluates conceptual quantum quantities 
+such as ELF and ELI on gridded fields based on quantum chemical wavefunctions."""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
```

Comment:
Seemingly no MPI support or any LAPACK dependencies. This can be dropped to `iccifort` `2019.1.144-GCC-8.2.0-2.31.1`